test/cmd/templates.sh

```diff
@@ -62,6 +62,7 @@ os::cmd::expect_success 'oc create -f examples/sample-app/application-template-d
 os::cmd::expect_success 'oc policy add-role-to-user admin test-user'
 os::cmd::expect_success 'oc login -u test-user -p password'
 os::cmd::expect_success 'oc new-project test-template-project'
+os::cmd::try_until_success 'oc policy who-can create imagestreams' 'test-user'
```

Comment:
doesn't matter, I'm just waiting until one of the permissions associated with being a project admin is reported